.github/ISSUE_TEMPLATE/0-bug-issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+---
+
```

Comment:
Hi Sherry,

Need a field explicitly: how to reproduce that. 

Is it possible for you to check the HSD and reuse some fields from HSD? Thanks a lot in advance!

Reply:
Yes, there is Issue behavior item below to reproduce the issue.